composer.json

```diff
@@ -45,7 +47,7 @@
     }
   },
   "scripts": {
-    "check-coding-standards": "vendor/bin/phpcs",
+    "check-coding-standards": "vendor/bin/phpcs -n -p --colors",
```

Comment:
why `-n`? Usually it's good to see and fix warnings.
I usually use `phpcs --parallel=8 -s` locally and `--runtime-set ignore_warnings_on_exit 1` in CI.

Reply:
right, and I needed to add cs and psalm to ci